app/[locale]/institute/administration/(committees)/committee.tsx

```diff
@@ -85,41 +92,46 @@ export default async function Committee({
             </TableHead>
           </TableRow>
         </TableHeader>
-
```

Comment:
This is fine though, since `<TableBody>` is now significantly smaller hence more digestable.